pkg/kubelet/kubelet_pods.go

```diff
@@ -166,6 +167,12 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 				return nil, cleanupAction, fmt.Errorf("volume subpaths are disabled")
 			}
 
+			// Expand subpath variables
+			if utilfeature.DefaultFeatureGate.Enabled(features.VolumeSubpathExpansion) {
+				mount.SubPath = kubecontainer.ExpandContainerVolumeMounts(mount, expandEnvs)
+				glog.V(6).Infof("Expanded Subpath Mount %s\n", mount.SubPath)
```

Comment:
Why adding `\n` at the end of the message?

Also, how meaningful is this log message -- it doesn't show the original pre-expanded path, nor does it identify the pod

Reply:
Does Infof put a cr/lf in the line?
Otherwise here I am only displaying what the expanded subpath is. There is already logging in this area which shows pod definitions etc. So I don't think it is worth repeating. But I can either take it out or add what you think is more valuable. When  I was debugging this, the only thing I cared about was the result of the expansion.  Do you think it needs more?

Reply:
Having looked at it, seems like it adds no value, so removed it